features/step_definitions/challege_gift_exchange_steps.rb

```diff
@@ -111,6 +111,17 @@
   step %{I open signups for "#{title}"}
 end
 
+Given /^the open gift exchange "([^\"]*)" has overlapping tags$/ do |challengename|
+  step %{I am logged in as "mod1"}
+  step "I have overlapping challenge tags setup"
+  step %{I set up the collection "#{challengename}" with name "#{challengename.gsub(/[^\w]/, '_')}"}
+  step %{I select "Gift Exchange" from "challenge_type"}
+  click_button("Submit")
```

Comment:
Is there a difference here that requires using `click_button` versus `step %{I submit}` like below?